### PR TITLE
Add docs requirements

### PR DESCRIPTION
**Pull request by mmuckley**
This adds a `requirements.txt` to help ReadTheDocs build the package documentation. The live site for this branch can be seen here: https://neuralcompression.readthedocs.io/en/mmuckley-add_docs_requirements/

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+sphinx>=3.2.0
+pytorch-sphinx-theme>=0.0.19
+sphinxcontrib-katex
```
Comment (0x00b1) [inline docs/requirements.txt:3]: `sphinxcontrib-katex>=0.8.6`

File: docs/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+sphinx-copybutton
```
Comment (0x00b1) [inline docs/requirements.txt:4]: `sphinx-copybutton>=0.4.0`

File: docs/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+sphinx>=3.2.0
+pytorch-sphinx-theme>=0.0.19
```
Comment (0x00b1) [inline docs/requirements.txt:2]: `pytorch_sphinx_theme @ git+https://github.com/pytorch/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme`

I believe this is not regularly released to PyPI.

**Comment by mmuckley** [comment]
All comments should be resolved.

**Review by QuentinDuval — APPROVED** [review]

**Pull request merged**